application/versions.txt

```diff
@@ -7,7 +7,8 @@ codeception/module-yii2                1.1.8              Codeception module for
 codeception/specify                    2.0.0              BDD code blocks for PHPUnit...
 codeception/verify                     2.3.0              BDD assertion library for P...
 fillup/fake-bower-assets               2.0.9              Use Composer "replace" to f...
```

Comment:
As an aside, perhaps something other than fillup/fake-bower-assets would be good to look for in another PR.

Reply:
Agreed (though low priority). I believe "asset-packagist" replaces the ned for that, now, fwiw.